ios/RNSScreenStackAnimator.mm

```diff
@@ -9,6 +9,9 @@
 static const float RNSSlideCloseTransitionDurationProportion = 0.25 / 0.35;
 static const float RNSFadeCloseTransitionDurationProportion = 0.15 / 0.35;
 static const float RNSFadeCloseDelayTransitionDurationProportion = 0.1 / 0.35;
+// same value is used in other projects using similar approach for transistions
+// and it looks the most similar to the value used by Apple
+constexpr float RNSShadowViewMaxAlpha = 0.1;
```

Comment:
Idk why I suggested sole `constexpr` here, let's make it `static` also, so that the definition is visible only in this translation unit.

```suggestion
static constexpr float RNSShadowViewMaxAlpha = 0.1;
```